GoogleCloudExtension/GoogleCloudExtension/StackdriverLogsViewer/LogsViewerToolWindowControl.xaml.cs

```
@@ -97,14 +62,13 @@ private void dataGrid_ScrollChanged(object sender, ScrollChangedEventArgs e)
             ScrollViewer sv = e.OriginalSource as ScrollViewer;
             if (sv == null)
             {
-                Debug.Assert(false);
                 return;
             }
 
-            DebugWriteLine($"e.VerticalOffset={e.VerticalOffset}, ScrollableHeight={sv.ScrollableHeight}, e.VerticalChange={e.VerticalChange}, e.ViewportHeight={e.ViewportHeight}");
+            Debug.WriteLine($"e.VerticalOffset={e.VerticalOffset}, ScrollableHeight={sv.ScrollableHeight}, e.VerticalChange={e.VerticalChange}, e.ViewportHeight={e.ViewportHeight}");
             if (e.VerticalOffset == sv.ScrollableHeight)
             {
-                DebugWriteLine("Now it is at bottom");
+                Debug.WriteLine("Now it is at bottom");
                 ViewModel?.LoadNextPage();
```

Is really a situation when `ViewModel` can be `null` and the program be well behaved? I think you should let it crash if `ViewModel` is `null`, otherwise you will never find the bug.